sdk/servicebus/service-bus/samples/typescript/gettingStarted/useProxy.ts

```diff
@@ -10,16 +10,21 @@ import { ServiceBusClient } from "@azure/service-bus";
 import WebSocket from "ws";
 import url from "url";
 import httpsProxyAgent from "https-proxy-agent";
+import { HttpsProxyAgentOptions } from "https-proxy-agent";
```

Comment:
Can we not combine the above 2 lines?

Reply:
Seems like we should be able to! I'll try and push in a bit.

Reply:
it works! pushing